### PR TITLE
feat: testing no reviewers

### PR DESCRIPTION
**Pull request by akarath**

### CODE REVIEW

File: test2.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Class Main
+end
```
Comment (jhasatyam) [inline test2.rb:2]: ok

Comment (jhasatyam) [reply test2.rb:2]: yo

**Review by jhasatyam — CHANGES_REQUESTED** [review]

File: test2.rb (hunk 1 above)
Comment (jhasatyam) [reply test2.rb:2]: ok

Comment (jhasatyam) [reply test2.rb:2]: coll

File: test2.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Class Main
```
Comment (akarath) [inline test2.rb:1]: I comment on I

**Review by jhasatyam — APPROVED** [review]

**Pull request merged**